src/equations/equations.jl

```diff
@@ -4,5 +4,11 @@ abstract type AbstractCompressibleMoistEulerEquations{NDIMS, NVARS} <:
               AbstractEquations{NDIMS, NVARS} end
 abstract type AbstractCompressibleEulerPotentialTemperatureEquations{NDIMS, NVARS} <:
               AbstractEquations{NDIMS, NVARS} end
+
+struct flux_LMARS{SpeedOfSound}
+    # Estimate for the speed of sound
+    speed_of_sound::SpeedOfSound
+end
```

Comment:
Can't we use the one from Trixi.jl directly?

Reply:
Oh thanks, I didn't think about it. Now the fluxes are using the struct field from Trixi directly, so there's no need to duplicate the definition.